packages/discord.js/typings/index.test-d.ts

```diff
@@ -2370,33 +2371,33 @@ expectType<
 >(NonThreadGuildBasedChannel);
 expectType<GuildTextBasedChannel>(GuildTextBasedChannel);
 
-const button = new ButtonBuilder({
+const button = new PrimaryButtonBuilder({
   label: 'test',
   style: ButtonStyle.Primary,
-  customId: 'test',
+  custom_id: 'test',
 });
 
 const selectMenu = new StringSelectMenuBuilder({
-  maxValues: 10,
-  minValues: 2,
-  customId: 'test',
+  max_values: 10,
+  min_values: 2,
+  custom_id: 'test',
 });
 
 new ActionRowBuilder({
   components: [selectMenu.toJSON(), button.toJSON()],
 });
 
 new StringSelectMenuBuilder({
-  customId: 'foo',
+  custom_id: 'foo',
 });
 
-new ButtonBuilder({
+new PrimaryButtonBuilder({
   style: ButtonStyle.Danger,
 })
-  .setEmoji('<a:foo:123>')
-  .setEmoji('<:foo:123>')
-  .setEmoji('foobar:123')
-  .setEmoji('😏')
+  .setEmoji(resolvePartialEmoji('<a:foo:123>'))
```

Comment:
This is merely testing the return type of resolveColor, so should just do that. The builder itself gets tested in /builders.